frontend/packages/knative-plugin/src/components/overview/EventSourceResources.tsx

```diff
@@ -15,11 +17,30 @@ import {
 import { PodModel } from '@console/internal/models';
 import { PodsOverview } from '@console/internal/components/overview/pods-overview';
 
-export type EventSinkServicesOverviewListProps = {
+export type EventSourceResourcesProps = {
   obj: K8sResourceKind;
+  ownedSources?: K8sResourceKind[];
 };
 
-const EventSinkServicesOverviewList: React.FC<EventSinkServicesOverviewListProps> = ({ obj }) => {
+export const OwnedSourcesList: React.FC<{ source: K8sResourceKind }> = ({ source }) => {
```

Comment:
It would be better to have one component per file. Moving this out to new a file and we can then add its own test cases.